src/components/widget.js

```diff
@@ -150,172 +145,168 @@ class Widget extends LitElement {
+          html`<img
+            alt=""
+            class="w-14 h-14 rounded-full"
+            src="${this.member.avatarUrl}"
+          />`}
+
+          <div class="flex flex-col">
+            <!-- Name -->
+            <span
+              class="block text-lg font-semibold leading-5 text-gray-900 truncate"
+              >${this.member.name}</span
+            >
+
+            <!-- Title -->
+            <span class="block text-sm leading-5 text-gray-500 truncate"
+              >${this.member.jobTitle}</span
+            >
+
+            <!-- Organization -->
+            ${!!this.member.organization
               ? html`
-            <div class="flex flex-row justify-start items-center mt-1">
-              ${
-                this.member.organization.logo_url &&
-                html`<img
-                  class="mr-1 w-5 h-5"
-                  src="${this.member.organization.logo_url}"
-                />`
+                  <div
+                    class="flex flex-row justify-start items-center mt-1 leading-5"
+                  >
+                    ${this.member.organization.logo_url &&
+                    html`<img
+                      alt=""
+                      class="mr-1 w-5 h-5"
+                      src="${this.member.organization.logo_url}"
+                    />`}
+                    <!-- If organisation doesn't include a protocol (ie https://), add one so it's treated as absolute -->
+                    <a
+                      href="${this.member.organization.website.match(
+                        /https?:\/\//
+                      )
+                        ? this.member.organization.website
+                        : `https://${this.member.organization.website}`}"
+                      target="_blank"
+                      rel="noreferrer"
+                      class="mr-2 text-sm font-semibold text-[#6C4DF6] hover:underline"
+                      >${this.member.organization.name}</a
+                    >
+                    ${this.member.organization.lifecycle_stage === "customer"
+                      ? html`<span class="sr-only">Customer</span>
+                          <span aria-hidden="true"
+                            >${unsafeSVG(iconCustomer)}</span
+                          >`
+                      : nothing}
+                  </div>
+                `
+              : // Just used for spacing when the organisation is not present
+                html`<div class="mt-2"></div>`}
+          </div>
+        </section>
+
+        <!-- Pills -->
+        <section
+          class="flex flex-row justify-start items-center mt-3 space-x-1"
+        >
+          ${this.member.teammate
+            ? html`<obe-pill
+                icon="${iconOrbitLevel}"
+                value="Teammate"
+              ></obe-pill>`
+            : html`<obe-pill
+                icon="${iconOrbitLevel}"
+                name="Orbit"
+                value="${this.member.orbitLevel || "N/A"}"
```

Comment:
Should we remove the pill if we don’t know the Orbit Level? Displaying `N/A` feels a bit off to me. What do you think?